guides/doc-Planning_for_Project/topics/Required_Technical_Users.adoc

```diff
@@ -25,14 +25,20 @@ Because of potential conflicts with local users that {Project} creates, you cann
 |User name |UID |Group name |GID |Flexible UID and GID |Home |Shell
 |foreman |N/A |foreman |N/A |yes |/usr/share/foreman |/sbin/nologin
 |foreman-proxy |N/A |foreman-proxy |N/A |yes |/usr/share/foreman-proxy |/sbin/nologin
+|apache |48 |apache |48 |no |/usr/share/httpd |/sbin/nologin
+|postgres |26 |postgres |26 |no |/var/lib/pgsql |/bin/bash
+ifdef::katello,orcharhino,satellite[]
 |pulp |N/A |pulp |N/A |no |N/A |/sbin/nologin
+endif::[]
 |puppet |N/A |puppet |N/A |yes |/opt/puppetlabs/server/data/puppetserver |/sbin/nologin
 |puppetserver |N/A |puppet |N/A |no |/opt/puppetlabs/server/data/puppetserver |/sbin/nologin
```

Comment:
There is no `puppetserver` user. The agent runs as root and the server runs as puppet.